packages/pyright-internal/src/analyzer/typePrinter.ts

```diff
@@ -245,11 +253,14 @@ export function printType(
                 // If it's a Callable with a ParamSpec, use the
                 // Callable notation.
                 const parts = printFunctionParts(type, printTypeFlags, returnTypeCallback, recursionTypes);
-                const paramSignature = `(${parts[0].join(', ')})`;
+                const paramSignature = annotationFormat ? `${parts[0].join(', ')}` : `(${parts[0].join(', ')})`; // no brackets if it's an annotation
```

Comment:
Please don't use end-of-line comments. In the pyright code base, comments should be block comments prior to the line, and they should be complete sentences starting with a capital letter engine with a period. I'm not sure this comment is needed since the intent is pretty obvious.